mod-finance/schemas/fy_finance_data.json

```diff
@@ -93,6 +93,10 @@
       "description": "The encumbrance percentage limit for this budget",
       "type": "number"
     },
+    "budgetAfterAllocation": {
+      "description": "The amount of the budget after the allocation",
+      "type": "number"
+    },
```

Comment:
If this field be provided for both Commit and Preview mode then sample file also should be updated. But in any case better to provide it in response for both modes

Reply:
Fixed it, thanks